deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml

```diff
@@ -12,16 +12,17 @@ metadata:
     description: Cloud-native, persistent storage for containers.
 spec:
   # replaces: 
-  displayName: Storageos Cluster Operator
-  description: |
-    StorageOS is a software-defined storage platform that transforms commodity
-    server or cloud based disk capacity into enterprise-class storage for
-    running persistent workloads such as databases in containers.
+  displayName: Storageos Cluster Operator description: |
```

Comment:
`description` should be in a new line.

Reply:
I think we discussed about the name and decided to name it just `StorageOS`.

Reply:
Nicely spotted.  Also changed name.